packages/api/db/migration/20230111164146_create_nomination_tables.js

```diff
@@ -0,0 +1,117 @@
+export const up = async function (knex) {
+  const now = knex.fn.now();
+  const litefarmUserId = 1;
+
+  await knex.schema.createTable('nomination_type', (table) => {
+    table.string('nomination_type').primary().notNullable();
```

Comment:
@Duncan-Brain strings cannot be a primary key

Reply:
we can not have duplicate strings. same string value should only appear once in a database. A single change in string value reference by an id indicates an updated to all foreign keys.

Reply:
Its a primary key so any attempt to create a duplicate will fail. Do you have any articles I can read saying this is a bad idea? I have personally read the opposite.

Reply:
I double checked because I am not 100% confident on this but trying to change it in PG results in this: 
![image](https://user-images.githubusercontent.com/30075973/211950937-cc1c78f3-cee2-4736-a21c-08bd850b2be4.png)

Reply:
@Duncan-Brain you review this https://online.umich.edu/courses/database-design-postgresql/